Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_InteractUse.cs

```diff
@@ -201,6 +203,10 @@ private void DoStartUseObject(object sender, ControllerInteractionEventArgs e)
 
             if (touchedObject != null && interactTouch.IsObjectInteractable(touchedObject))
             {
+                var vrobj = touchedObject.GetComponent<VRTK_InteractableObject>();
```

Comment:
OK, I surrender, it will be named "interactableObjectScript".
I'm a bit scared by this severity, anyway I got you: if many people work on the same code they must be able to clearly undestand what they read.
I already changed those names in my Working Copy.
Let me know when you finish your review, so I can try to update my branch, anyway I will push in the next days another branch, so you can start merging this PR anyway or wait for the next PR (yeah... I am still scared by Git!).